docs/_config.yml

```diff
@@ -77,12 +77,7 @@ defaults:
       layout: plain
       nav-pos: 99999 # Move to end if no pos specified
 
-markdown: KramdownPygments
```

Comment:
Some questions regarding the removed lines:
- What is used for markdown and code highlighting now?
- Do we still support GitHub-flavored Markdown?
- What happens to new lines?

I didn't have time to build the docs again to check yet.

Reply:
* Kramdown for Markdown, Rouge for Code Highlighting.
* Yes
* Same behavior as before

The reason some of this is removed is because they're the [defaults](https://jekyllrb.com/docs/configuration/#default-configuration).